data/crac-io/crac-io-json/src/test/java/com/powsybl/openrao/data/craciojson/JsonRetrocompatibilityTest.java

```diff
@@ -309,6 +309,16 @@ void importV2Point4Test() {
         testContentOfV2Point4Crac(crac);
     }
 
+    @Test
+    void importV2Point5Test() {
+        // Add support for contingency in OnFlowConstraintInCountry
```

Comment:
modify the comment to reflect the change in the new crac version

Reply:
done